language/English/strings.po

```diff
@@ -592,6 +600,30 @@ msgctxt "#32218"
 msgid "Maximum supported protocol"
 msgstr ""
 
+msgctxt "#32220"
+msgid "Bad password!"
+msgstr ""
+
+msgctxt "#32221"
+msgid "The entered password is too weak.[CR]SSH root password is unchanged."
```

Comment:
"The entered password is too weak.[CR]SSH password is unchanged."

Reply:
OK